changelog/22.0/22.0.0/summary.md

```diff
@@ -59,6 +61,9 @@ The following fields can be dynamically changed -
 
 To upgrade to the newer version of the configuration file, first switch to using the flags in your current deployment before upgrading. Then you can switch to using the configuration file in the newer release.
 
+### <a id="last_insert_id"/>Added support for LAST_INSERT_ID(expr)
+
+Added support for LAST_INSERT_ID(expr) to align with MySQL behavior, enabling session-level assignment of the last insert ID via query expressions.
```

Comment:
It would be good to link to the PR and/or Issue so there is more context for those wanting to learn more.

```suggestion
### <a id="last_insert_id"/>Added support for LAST_INSERT_ID(expr)

Added support for `LAST_INSERT_ID(expr)` to align with MySQL behavior, enabling session-level assignment of the last insert ID via query expressions. For more information about this change see [#17295](https://github.com/vitessio/vitess/pull/17295).
```